app/services/preview_service.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 
 class PreviewService < ApplicationService
-  def initialize(edition)
+  def initialize(edition, optional_params = {})
```

Comment:
Do we anticipate to need to use these for any purpose other than republish? might be more explicit to have a boolean keyword argument of republish.